appveyor.yml

```diff
@@ -5,12 +5,18 @@ branches:
   only:
     - l10n_master # The branch that Crowdin targets when updates are made on .po files
     - master      # The branch that is published on Nuget.org
+    - l10n-pack   # For testing purpose
```

Comment:
This branch was for testing purpose only, I think it's fine to remove it, but I'm not sure if the CI/CD will kicks in